lambdas/common/models.py

```diff
@@ -0,0 +1,37 @@
+from __future__ import annotations
+
+import json
+from dataclasses import asdict, dataclass
+
+
+@dataclass
+class GranuleProcessingEvent:
+    """Event message for granule processing jobs"""
+
+    granule_id: str
+    attempt: int = 0
+
+    def to_envvar(self) -> dict[str, str]:
+        """Convert this event to environment variable"""
```

Comment:
Can you clarify this? What is expecting an env var as a dict (json)? I see that your stubbed out lambda handlers have this class imported, but why would they expect such a value as an env var rather than an event parameter?